uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -110,14 +138,18 @@ var Promise = require('es6-promise').Promise;
    * The content is the mix of the config template and the list of scenarios
    * under the scenarios/ folder
    *
-   * @return {string}
+   * @param  {Object} contactRolesAndIDsMap
+   * @return {String}
    */
-  function tempFileContent () {
+  function tempFileContent (contactRolesAndIDsMap) {
     var config = JSON.parse(fs.readFileSync(BACKSTOP_DIR + FILES.config));
     var content = JSON.parse(fs.readFileSync(BACKSTOP_DIR + FILES.tpl));
 
     content.scenarios = scenariosList().map(function (scenario) {
-      scenario.url = config.url + '/' + scenario.url;
+      scenario.url = config.url + '/' +
```

Comment:
this one-liner is too much. split things

Reply:
Done.